### DISCUSSION TITLE
Can the vivado project generated by the finn compiler be transplanted to the VCU108 board?

### DISCUSSION THREAD

**Discussion by wz19980621** [Xilinx/finn#898] [Q&A] [answered]
Can the vivado project generated by the finn compiler be transplanted to the VCU108 board?

**Comment by fpjentzsch** [comment, marked as answer]
Hi, this should be possible, see https://github.com/Xilinx/finn-examples/issues/65#issuecomment-1759450118

**Reply by wz19980621** [reply]
Thankyou so much
